--- origsrc/kcheckers-0.8.1/common.h	2005-12-29 11:53:16.000000000 -0600
+++ src/kcheckers-0.8.1/common.h	2008-07-14 09:51:28.093750000 -0500
@@ -35,7 +35,7 @@
 
 /* !!! Do not change PREFIX variable name, please. !!! */
 /* !!! It is used in kcheckers.pro. !!! */
-#define PREFIX		"/usr/local"
+#define PREFIX		"/usr"
 #define USER_PATH	".kcheckers"		// in $HOME
 #define THEME_DIR	"themes/"
 
--- origsrc/kcheckers-0.8.1/kcheckers.pro	2005-12-29 11:55:55.000000000 -0600
+++ src/kcheckers-0.8.1/kcheckers.pro	2008-07-14 10:00:15.781250000 -0500
@@ -21,8 +21,7 @@
 
 
 TARGET		= kcheckers
-PREFIX		= $$system(grep 'define PREFIX' common.h | cut -d'"' -f2)
-SHARE_PATH	= $$system(grep 'define SHARE_PATH' common.h | cut -d'"' -f2)
+PREFIX		= /usr
 
 TRANSLATIONS	= i18n/kcheckers_de.ts i18n/kcheckers_fr.ts
 #		i18n/kcheckers_ru.ts
